apps/sports-helsinki/src/pages/search/map.tsx

```diff
@@ -1,17 +1,22 @@
 import type { MapItem, AppLanguage } from 'events-helsinki-components';
 import {
+  FooterSection,
+  NavigationContext,
   useLocale,
   getURLSearchParamsFromAsPath,
   Navigation,
   MatomoWrapper,
   getLanguageOrDefault,
+  // eslint-disable-next-line import/no-unresolved
```

Comment:
This doesn't look right. This should not be needed. events-helsinki-components is very much needed and it must be resolvable.